### PR TITLE
Micnovak/example clean

### PR DESCRIPTION
**Pull request by micnovak**
cleaning of examples - first example uses only one path with generated sources
updated maven dependencies
updated README.md
created CHANGES.md
removed unused files
version bumped to 1.1.0-SNAPSHOT

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 group                  =  com.tailf.jnc
 archivesBaseName       =  project.name
-version                =  1.0
+version                =  1.0.1
 author                 =  Tail-f Systems
```
Comment (jojohans) [inline gradle.properties:4]: Related to my comment on README.md, when should we bump the version number?

**Review by jojohans — APPROVED** [review]

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+## Release History
+
+2022-06-22
+- Ganymed SSH-2 was replaced with SSHJ. Build system updated to Gradle. 
+
```
[the remark this answers is not in the document]
Comment (micnovak) [reply CHANGES.md:5]: I have replaced with `Next version (SNAPSHOT)`

File: README.md [hunk 3]
```diff
@@ -5,69 +5,75 @@ Java NETCONF Client.
...
-and converter in python. You need an installation of pyang to use the JNC
-plugin. Get it here: http://code.google.com/p/pyang/
+JNC (Java NETCONF Client) consists of two parts: 
 
-The JNC plugin can be used to generate Java class hierarchies from YANG data
+* Java library for NETCONF client
+* Pyang  plugin. You need an installation of pyang to use the JNC
+plugin. Get it here: https://github.com/mbj4668/pyang 
+(or use `pip3 install pyang`).
+
```
[the remark this answers is not in the document]
Comment (micnovak) [reply README.md:16]: Originally, google code version was referenced, so I assume it is "so old".

File: gradle.properties (hunk 1 above)
Comment (micnovak) [reply gradle.properties:4]: Ok, in that case i suggest we use 1.1.0-SNAPSHOT

https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN401

Comment (jojohans) [reply gradle.properties:4]: Fine with me.

**Pull request merged**